aip-console-tools-core/src/main/java/com/castsoftware/aip/console/tools/core/dto/ApplicationCommonDetails.java

```diff
@@ -0,0 +1,14 @@
+package com.castsoftware.aip.console.tools.core.dto;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Getter
+@Setter
+public class ApplicationCommonDetails {
```

Comment:
No meaning in having dto with just one member
Use Builder.Default

Reply:
These details might increase going down the line. So just kept it for now.